packages/popover/src/popover.tsx

```diff
@@ -76,11 +76,21 @@ if (__DEV__) {
   PopoverAnchor.displayName = "PopoverAnchor"
 }
 
+export interface PopoverTriggerProps {
+  /**
+   * The React child to use as the
+   * trigger for the popover
+   */
+  children: React.ReactChild
```

Comment:
Note: you might want to use ReactNode if you want to allow all valid children types such as fragments:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/274907dfb7bc98f63fe348defdb0c62774d4483b/types/react/index.d.ts#L221-L229

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/274907dfb7bc98f63fe348defdb0c62774d4483b/types/react/index.d.ts#L784

Reply:
Thanks, @stefee, change is made.